### PR TITLE
Support for TSLint processor

### PR DESCRIPTION
**Pull request by damianszczepanik**

### CODE REVIEW

File: src/main/java/pl/touk/sputnik/processor/tslint/TSLintScript.java [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+     * @param commands
+     *            arguments that will be used for TSLint
+     * @return results converted to string
+     * @throws ReviewException
+     *             when TSlint could not be executed
+     */
+    private String executeTSLint(String[] commands) throws ReviewException {
+        try {
+            log.info("Running native process: " + StringUtils.join(commands, " "));
+            ProcessBuilder builder = new ProcessBuilder(commands);
```
Comment (pjagielski) [inline src/main/java/pl/touk/sputnik/processor/tslint/TSLintScript.java]: I don't think that using sputnik to execute another node process is a good idea. Maybe it's possible to just pass the output of tslint to sputnik directly (e.g. as some runtime parameter?). @SpOOnman wdyt?

**Comment by SpOOnman** [comment]
@pjagielski it's not the best idea. We use Mozilla Rhino already but I think it won't be enough for TSLint?

**Comment by pjagielski** [comment]
We finally decided to allow external tool execution. I merged your branch with master and added some abstraction over executing external commands - see #139

**Comment by damianszczepanik** [comment]
After one year :-)
anyway - better than keeping this unresolved!

**Pull request closed without merging**